testing/deployer/sprawl/tls.go

```diff
@@ -113,6 +114,29 @@ fi
 		if err != nil {
 			return fmt.Errorf("could not create all necessary TLS certificates in docker volume: %v", err)
 		}
+
```

Comment:
This just extracts the contents of the CA public key so that we can use it later to dial gRPC on the servers to do resource operations.

Reply:
comment the code my friend